provider/version/version.go

```diff
@@ -24,7 +24,7 @@ import "fmt"
 var (
 	// Version is a string representing the current version of the Dynatrace Terraform provider.
 	// This should be in `major.minor.patch` format, with prereleases having an optional suffix introduced by a dash, for example, `1.2.0-beta`.
-	Version = "(unknown-version)"
+	Version = "1.87.1"
```

Comment:
I don't think this change was intended, right? 😅

Reply:
😅 no, not intended